config-model/src/main/java/com/yahoo/vespa/documentmodel/DocumentSummary.java

```diff
@@ -118,14 +116,11 @@ public String toString() {
         return "document-summary '" + getName() + "'";
     }
 
-    public void validate(DeployLogger logger) {
+    public void validate() {
         for (var inheritedName : inherited) {
             var inheritedSummary = owner.getSummary(inheritedName);
-            if (inheritedSummary == null) {
-                // TODO Vespa 9: Throw IllegalArgumentException instead
-                logger.logApplicationPackage(Level.WARNING,
-                                             this + " inherits '" + inheritedName + "' but this is not present in " + owner);
-            }
+            if (inheritedSummary == null && ! inheritedName.equals("default"))
```

Comment:
Inheriting 'default' does not make sense. I suggest logging a warning for now. Then we we might be able to remove the if later on.

Reply:
Sounds like a good idea, done